packages/vitepress-theme/src/index.scss

```diff
@@ -0,0 +1,61 @@
+@use "../../sit-onyx/src/styles/index.scss";
+@use "mixins.scss";
+
+// override default VitePress CSS variables to apply Onyx theming
+// see: https://github.com/vuejs/vitepress/blob/main/src/client/theme-default/styles/vars.css#L178
+:root {
+  // color palette
+  @include mixins.define-vitepress-color(brand, --onyx-color-brand-primary);
+  @include mixins.define-vitepress-color(tip, --onyx-color-surface-info);
+  @include mixins.define-vitepress-color(warning, --onyx-color-surface-warning);
+  @include mixins.define-vitepress-color(danger, --onyx-color-surface-danger);
+  @include mixins.define-vitepress-color(default, --onyx-color-surface-brand-neutral, -200);
+
+  // background
+  --vp-c-bg: var(--onyx-color-surface-bg-tinted);
+  --vp-c-bg-alt: var(--onyx-color-surface-bg-blank);
+  --vp-c-bg-elv: var(--onyx-color-surface-bg-tinted);
+  --vp-c-bg-soft: var(--onyx-color-surface-bg-blank);
+
+  // borders
+  // used for separators (e.g. below h2 headline)
+  --vp-c-border: var(--onyx-color-border-brand-neutral-300);
+  // border on interactive components (e.g. button outline)
+  --vp-c-divider: var(--onyx-color-border-brand-neutral-200);
+  // divide components in the page (e.g. header and page content)
+  --vp-c-gutter: var(--onyx-color-border-brand-neutral-200);
+
+  // typography
+  --vp-c-text-1: var(--onyx-color-text-body-paragraph-intense); // regular text
+  --vp-c-text-2: var(--onyx-color-text-body-paragraph-medium); // muted text (e.g. inactive menu)
+  --vp-c-text-3: var(--onyx-color-text-body-paragraph-soft); // subtle text (e.g. placeholders)
+  --vp-font-family-base: var(--onyx-font-family);
+  --vp-font-family-mono: var(--onyx-font-family-mono);
+
+  @include mixins.home-page-gradients(
+    var(--onyx-color-brand-primary-500),
+    var(--onyx-color-text-info-500)
+  );
+
+  // remove the following lines when this VitePress PR is merged:
+  // https://github.com/vuejs/vitepress/pull/3434
```

Comment:
it is merged ;)

Reply:
Merged but not release yet :D We need to wait for version `1.0.0-rc.37`

Reply:
Follow up ticket: #117